sklearn/tests/test_import_deprecations.py

```diff
@@ -28,6 +28,10 @@
     ('sklearn.cluster.mean_shift_', 'MeanShift'),
     ('sklearn.cluster.optics_', 'OPTICS'),
     ('sklearn.cluster.spectral', 'SpectralClustering'),
+
+    ('sklearn.mixture.base', 'BaseMixture'),
```

Comment:
This was not in the `__init__.py`, why are we having it then?

Reply:
because we still support importing from `sklearn.mixture.base` for the next 2 versions

Reply:
does this mean that we cannot use them after the next 2 versions? it is because the security issue or something?

Reply:
it means you can still import them from `sklearn.mixture._base`, but they're not a part of the public API and may change w/o a deprecation cycle. So your code may break between sklearn releases.

Reply:
Thank you very much!